src/i18n/strings/bn_IN.json

```diff
@@ -1,3 +1 @@
-{
-    "Riot is not supported on mobile web. Install the app?": "মোবাইল ওয়েব রায়ট সমর্থন করে না. অ্য়াপ টি ইনস্টল করতে চান?"
-}
+{}
```

Comment:
Should we remove the empty file?

Reply:
I'm tempted not to only because weblate looks to care very much about its files.

Reply:
Okay, fair enough.